styles/style.css

```diff
@@ -162,39 +202,65 @@ p {
 /* Article */
 
 .article {
```

Comment:
Could you position the two articles side by side like the picture from the assignment brief? Using flexbox may help here.
![Screenshot 2022-09-14 at 23 02 05](https://user-images.githubusercontent.com/67797986/190270900-754a9ff4-72c0-4728-b691-65ef6fb0d0c5.png)